lib/slack.ex

```diff
@@ -157,9 +165,94 @@ defmodule Slack do
     end
   end
 
+  @doc ~S"""
+  Turns a string like `"@USER_NAME"` into the ID that Slack understands (`"U…"`).
+  """
+  def lookup_user_id("@" <> user_name, slack) do
```

Comment:
Thoughts on extracting these functions into their own module?

Potential name(s): `Slack.Lookups`, `Slack.Lookup` or something better because I'm not completely sold on my suggestions.